containers/SideNav/SideNav.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react';
+import { createContainer } from 'unstated-next';
+
+import { SubMenuLink } from 'sections/shared/Layout/constants';
+
+export type SubMenuConfiguration = {
+	routes: SubMenuLink[] | null;
+	topPosition: number;
+};
+
+export default createContainer(Container);
+
+function Container() {
+	const [isShowingMobileSideNav, setIsShowingMobileSideNav] = useState(false);
+	const [subMenuConfiguration, setSubMenuConfiguration] = useState<SubMenuConfiguration>({
+		routes: null,
+		topPosition: 0,
+	});
+	const isShowingSubMenu = !!subMenuConfiguration?.routes ?? false;
```

Comment:
if subMenuConfiguration.routes is null/undefined won't `!!` already make it false?

Reply:
Gotcha 👍🏽